sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/data/FollowingViewportStateOptions.kt

```diff
@@ -13,85 +13,85 @@ class FollowingViewportStateOptions private constructor(
    *
    * Defaults to `16.35`.
    */
-  val defaultZoom: Double,
+  val zoom: Double,
   /**
    * The default pitch that will be generated for following camera frames.
    *
    * Defaults to `45.0` degrees.
```

Comment:
Move `45.0` to const and add reference here (as well as default value in builder below). Applies to zoom as well